modular_skyrat/code/modules/projectiles/guns/ballistic/shotgun.dm

```diff
@@ -63,3 +63,119 @@
 /obj/item/gun/ballistic/revolver/doublebarrel/upgraded/sawoff(mob/user)
 	to_chat(user, "<span class='warning'>Considering the modifications, sawing it off probably would break it entirely.</span>")
 	return
+
+//
+//
+//
+//
+////////////////////////
+// IMPROVISED SHOTGUN //
+////////////////////////
+//		Let's throw out the old and bring in a break-action shotgun.
+
+/obj/item/gun/ballistic/shotgun/improvised
+	name = "improvised shotgun"
+	desc = "A break-action 12 gauge shotgun. You need both hands to fire this."
```

Comment:
https://github.com/Skyrat-SS13/Skyrat13/pull/3346
Yes this is shilling, but my PR would help a lot with this kind of balance situation